labelbox/schema/task.py

```diff
@@ -31,6 +34,7 @@ class Task(DbObject):
     name = Field.String("name")
     status = Field.String("status")
     completion_percentage = Field.Float("completion_percentage")
+    result = Field.String("result")
```

Comment:
maybe call this result_url

Reply:
There's already a resolver called `result` off Task, so I think it's best to keep the naming consistent